frontend/app/src/people/utils/AssignBounty.tsx

```diff
@@ -35,6 +36,37 @@ export default function AssignBounty(props: ConnectCardProps) {
     setToasts([]);
   };
 
+  async function startPolling(paymentRequest: string) {
```

Comment:
I see we do this same polling logic in a few places can we make this a common function? there seems to be some differences but maybe we can do a followup PR to make this common logic